buildspec.yml

```diff
@@ -0,0 +1,17 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+      - ./gradlew build packageLambda
```

Comment:
The canonical target for verifying the build is `gradle check`, which also does static analysis.

Reply:
should the `build` is the full build, including `check` task?